packages/core/src/components/MapContainer.vue

```diff
@@ -158,7 +158,7 @@ export default Vue.extend({
     }
     this.updateListeners(this.hasWindowSize)
     this.setConfiguration(this.mapConfiguration)
-    this.mapConfiguration.locales?.forEach?.((lng: LanguageOption) =>
+    this.mapConfiguration.locales?.forEach?.((lng: Locale) =>
       i18next.addResourceBundle(lng.type, 'common', lng.resources, true)
```

Comment:
```suggestion
    this.mapConfiguration.locales?.forEach?.((locale: Locale) =>
      i18next.addResourceBundle(locale.type, 'common', locale.resources, true)
```
🎩 if it seems irrelevant to you